source/reference/beta-program/namespace-remapping.txt

```diff
@@ -12,4 +12,128 @@ Change Collection Namespaces
    :depth: 2
    :class: singlecol
 
-.. include:: /includes/beta-feature.rst
+.. include:: /includes/beta-feature
+
+Starting in ``mongosync`` beta 1.8, you can remap database names during sync.
+This allows you to take data in one database on the source cluster and migrate
+it into a different database on the destination cluster.
+
+Syntax
+------
+
+The :ref:`/start <c2c-api-start>` API endpoint accepts an optional ``namespaceRemap``
+parameter with the following syntax:
+
+.. code-block:: sh
+
+   curl <host>:<port>/api/v1/start -XPOST \
+   --data '
+      {
+         "source": "cluster0",
+         "destination": "cluster1"
+         "namespaceRemap": [
+            {
+               "from": {
+                  "database": "<source-database>"
+               },
+               "to": {
+                  "database": "<destination-database>"
+               }
+            }
+         ]
+      } '
+
+Parameter Fields
+----------------
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 20 13 13 54
+
+   * - Field
+     - Type
+     - Required
+     - Description
+   * - ``namespaceRemap``
+     - array
+     - Optional
+     - Array of documents that specify namespace changes to make during sync.
+
+       .. versionadded:: 1.8 Beta
+
+   * - ``namespaceRemap[n].`` ``from``
+     - document
+     - Required
+     - Document that specifies the database on the source cluster
+       to migrate from in the remapping operation.
+
+       .. versionadded:: 1.8 Beta
+
+   * - ``namespaceRemap[n].`` ``from.database``
+     - string
+     - Required
+     - Database to read from on the source cluster.
+
+       .. versionadded:: 1.8 Beta
+
+   * - ``namespaceRemap[n].`` ``to``
+     - document
+     - Required
+     - Document that specifies the database on the destination cluster
+       to write to in the remapping operation.
+
+       .. versionadded:: 1.8 Beta
+
+   * - ``namespaceRemap[n].`` ``to.database``
+     - string
+     - Required
+     - Database to write to on the destination cluster.
```

Comment:
Same comment. I don't feel super strongly. If we want to keep read/write instead of migrate to/from. Then maybe we should make the ``namespaceRemap[n].from`` use bullet use "read" too.

Reply:
Fixed